HousingRepairsOnlineApi/Gateways/IBlobStorageGateway.cs

```diff
@@ -1,9 +1,11 @@
-﻿using System.Threading.Tasks;
+﻿using System;
+using System.Threading.Tasks;
 
 namespace HousingRepairsOnlineApi.Gateways
 {
     public interface IBlobStorageGateway
     {
         Task<string> UploadBlob(string base64Img, string fileExtension);
+        Task<string> GetServiceSasUriForBlob(string fileName, int daysUntilExpiry, string storedPolicyName = null);
```

Comment:
The `fileName` parameter would confuse callers of this method as blob storage doesn't use file names.
Please rename to something more appropriate, i.e. blobName
```suggestion
        Task<string> GetServiceSasUriForBlob(string blobName, int daysUntilExpiry, string storedPolicyName = null);
```